shared/dto/widgets/base-widget-data.interface.ts

```diff
@@ -1,12 +1,9 @@
+import { BaseWidget } from '@shared/dto/widgets/base-widget.entity';
+
 /**
  * Represents an array of widget data objects.
  */
 export type WidgetData = Array<{
-  /**
-   * Unique identifier
-   */
```

Comment:
as mentioned, we won't have any ID available here, unless we proactively compute it